test/system/trader.js

```diff
@@ -0,0 +1,179 @@
+const cryptoClient = require('../../lib/cryptonite-client');
+const logInitialDetails = require('./initilaLogger');
+
+class Trader {
+
+  constructor(config) {
+    this.config = config;
+    this.client = new cryptoClient(this.config);
+    this.market = 'LTC/BTC';
+    this.init_state(this.config);
+  }
+
+  init_state() {
```

Comment:
It's probably worth returning the Promise from this and the other functions.
Unlike the Simulator, the System Test's Trader will be running a deterministic sequence of operations coordinated at the top level (in index.js). It will be useful for the coordination code to know when the things it has called are complete.